mosquitto/DOCS.md

```diff
@@ -78,7 +78,8 @@ A file containing the private key. Place this file in the Home Assistant `ssl` f
 
 ### Option: `require_certificate`
 
-If set to `true` encryption will be enabled using the cert- and keyfile options.
+If set to `true`, the client must provide a valid certificate in order to connect successfully.  
+If set to `false`, the SSL/TLS component of the client will verify the server but there is no requirement for the client to provide anything for the server, authentication is limited to the MQTT username/password.
```

Comment:
```suggestion
If set to `true`, require a client certificate. Means encrypted connections are required, the client must verify and trust the server's certificate and must provide a certificate signed by the CA in `cafile`.
If set to `false`, do not require a client certificate. Can still enable encrypted connections using the `certfile` and `keyfile` options.
```
Good idea, took another stab at it though. It's more then just requiring a client certificate, this option also means only encrypted connections are possible and both parties must trust and verify each others' certificate. Also the client certificate must be signed by the CA in `cafile`. The `cafile` option doesn't do anything if `require_certificate` is false.